### PR TITLE
Use flags for executor settings

### PR DESCRIPTION
**Pull request by tempoz**
Straightforward replacement of executor config with flags.

---

**Version bump**: Minor

### CODE REVIEW

**Comment by tempoz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1771** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1771" target="_blank">Graphite</a> 
    * **PR #1772** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1772" target="_blank">Graphite</a> 
      * **PR #1773** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1773" target="_blank">Graphite</a> 
        * **PR #1774** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1774" target="_blank">Graphite</a> 
          * **PR #1775** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1775" target="_blank">Graphite</a> 
            * **PR #1776** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1776" target="_blank">Graphite</a> 
              * **PR #1777** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1777" target="_blank">Graphite</a> 
                * **PR #1778** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1778" target="_blank">Graphite</a> 
                  * **PR #1779** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1779" target="_blank">Graphite</a> 
                    * **PR #1780** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1780" target="_blank">Graphite</a> 
                      * **PR #1783** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1783" target="_blank">Graphite</a> 
                      * **PR #1833** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1833" target="_blank">Graphite</a> 
                        * **PR #1834** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1834" target="_blank">Graphite</a> 
                          * **PR #1835** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1835" target="_blank">Graphite</a> 
                            * **PR #1836** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1836" target="_blank">Graphite</a> 
                              * **PR #1837** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837" target="_blank">Graphite</a>  (this PR)
                                * **PR #1838** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1838" target="_blank">Graphite</a> 
                                  * **PR #1839** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1839" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).

File: enterprise/server/remote_execution/executor/config/config.go
Comment (tylerwilliams) [inline enterprise/server/remote_execution/executor/config/config.go:10]: Is this an intermediate CL or why do we want these here instead of in the various libraries like we're doing with the app flags?

**Review by tylerwilliams — APPROVED** [review]

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: LYGXjCZ9jzXE0ra5XapW

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: bO0bzB7R9g7bK8rQ9uPT

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: x9vSQMZuwVwdzOiJU1Ph

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: xLhUv52COpx6k7AsRAsn

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: KZCXEHPsl7FVcdrah9Au

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1837?utm_source=stack-comment).


Job Reference: eJKjgKqVU3dUX0WhID3h

**Pull request merged**
